css-animations-2/Overview.bs

```diff
@@ -845,29 +845,80 @@ intended to support using [=triggers=] for <em>other</em> purposes in the future
 For now, though, [=triggered animations=] are the only user of this feature.
 
 
-<h3 id="trigger-scope">
-Trigger Scope/Resolution</h3>
+<h3 id="trigger-name-scoping">
+Trigger Name Scoping</h3>
 
-All [=triggers=] are document-global by default,
-similar to <l spec="css-anchor-position-1">[=anchor names=]</l>.
+A <dfn export>trigger-instantiating property</dfn> specifies the details for
+instantiating an [=Animation Trigger=] along with a name for the instantiated
+trigger, a <dfn export>trigger name</dfn>.
+
+All [=trigger names=] are document-global by default,
+similar to <l spec="css-anchor-position-1">[=anchor names=]</l>, but may be
+scoped by [[#trigger-scope| trigger-scope]].
+
+ISSUE: Is "document-global" the right term here given that these "global" names are
+still tree scoped?
 
 If a single element attempts to define multiple [=triggers=] of different types
-with the same name,
-it only exposes one of them,
+with the same [=trigger name=],
+it only exposes one of the [=triggers=],
 with [=event triggers=] winning over [=timeline triggers=].
 
 Note: This order is completely arbitrary
 (based on alphabetic order of the concept name),
 as this is just an error case.
 
-If multiple elements define [=triggers=] with the same name,
+If multiple elements define [=triggers=] with the same [=trigger name=],
 the [=trigger=] defined by the later element in [=tree order=] is used.
 
-Note: This behavior will be improved by a <css>trigger-scope</css> property,
-not yet defined,
-letting you define triggers that are only visible to subtrees
-and references that only search in that subtree
-(just like 'anchor-scope').
+<h4 id="trigger-scope">
+Scoping Trigger Names: the 'trigger-scope' property</h4>
+
+<pre class=propdef>
+Name: trigger-scope
+Value: none | all | <<dashed-ident>>#
+Initial: none
+Applies to: all elements
+Inherited: no
+Animation type: discrete
```

Comment:
Given that resolving to a different trigger could result in starting an animation, I think this may need to be "not animatable".